README.MD

```diff
@@ -405,6 +406,26 @@ If you have trouble running any example try the following:
 
 </details>
 
+<details>
+    <summary>"A system issue occured, Please try again" when sending SMS</summary>  
+<br/>
+
+A. Check your `FacebookAppID` and `AccountKitClientToken` on iOS `Info.plist` and Android `strings.xml` is correct  
+
+B. If you have enabled the **client access token flow in fb account kit dashboard**, then `configure` `responseType` should be set to `code`.
```

Comment:
Maybe the following is more clear?
_then the `responseType` should be set to `code` when calling `configure`_